client/my-sites/site-settings/date-time-format/time-format-option.jsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ExternalLink from 'components/external-link';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormInput from 'components/forms/form-text-input';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import { phpToMomentDatetimeFormat } from 'lib/formatting';
+import { defaultTimeFormats } from './default-formats';
+
+export const TimeFormatOption = ( {
+	disabled,
+	isCustom,
+	localizedDate,
+	setCustomTimeFormat,
+	setTimeFormat,
+	timeFormat,
+	translate,
+} ) => (
+	<FormFieldset>
+		<FormLabel>
+			{ translate( 'Time Format' ) }
+		</FormLabel>
+		{ defaultTimeFormats.map( ( format, index ) =>
+			<FormLabel key={ index }>
+				<FormRadio
+					checked={ ! isCustom && format === timeFormat }
+					disabled={ disabled }
+					name="time_format"
+					onChange={ setTimeFormat }
+					value={ format }
+				/>
+				<span>{ localizedDate.format( phpToMomentDatetimeFormat( format ) ) }</span>
+			</FormLabel>
+		) }
+		<FormLabel className="date-time-format__custom-field">
+			<FormRadio
+				checked={ isCustom }
+				disabled={ disabled }
+				name="time_format"
+				onChange={ setCustomTimeFormat }
+				value={ timeFormat }
+			/>
+			<span>
+				{ translate( 'Custom', { comment: 'Custom date/time format field' } ) }
+				<FormInput
+					disabled={ disabled }
+					name="time_format_custom"
+					onChange={ setCustomTimeFormat }
+					type="text"
+					value={ timeFormat || '' }
+				/>
+				<FormSettingExplanation>
+					{ isCustom && timeFormat
+						? translate( 'Preview: %s', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **67** times:
`translate( 'Preview %s' )` *ES Score: 18*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).